modules/ROOT/pages/enable-verbose-logging.adoc

```diff
@@ -0,0 +1,68 @@
+= Enable Verbose Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can enable verbose logging for different Modules and Connectors. Because of the high impact caused in application performance, verbose logging must be enabled only for testing and troubleshooting purposes.
+
+== Configure Verbose Logging
+
+To configure verbose logging, add an entry to your `log4j2.xml` indicating the package names of the Connectors or Modules you want to log, and set the log level to `DEBUG`.
+
+The following example shows how to find the package names for the JMS Connector and configure verbose logging:
```

Comment:
The following example shows how to find the package names **for Anypoint Connector JMS (JMS Connector) and** configure verbose logging: